lib/analytics.js

```diff
@@ -114,13 +126,18 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
     if (!Integration) delete settings[name];
   }, settings);
 
-  // add integrations
-  each(function(opts, name) {
-    var Integration = self.Integrations[name];
-    var integration = new Integration(clone(opts));
-    self.log('initialize %o - %o', name, opts);
-    self.add(integration);
-  }, settings);
+  if (doNotTrack) {
+    // Clear manually added integrations
+    this._integrations = {};
```

Comment:
> but the msg.context.doNotTrack being set on 692 would represent the value that the customer has chosen not the segment project owner.

correct. it doesn't make sense to send the project level setting as that's pretty constant and doesn't change from user to user.

ALso, yup it's slightly confusing 2 different settings are named the same thing.

And yes it's sort of like `context.device.adTrackingEnabled` except it means something else, and has the ability to disable events entirely.